whois-commons/src/main/java/net/ripe/db/whois/common/dao/jdbc/JdbcRpslObjectOperations.java

```diff
@@ -493,6 +511,27 @@ public SerialEntry mapRow(ResultSet rs, int rowNum) throws SQLException {
         }, serialId);
     }
 
+    private static List<SerialEntry> getSerialEntryWithBlobsFromLastForNrtm4(final JdbcTemplate jdbcTemplate) {
+        return jdbcTemplate.query("" +
+            "SELECT serials.serial_id, "+
+            "       serials.operation, " +
+            "       serials.atlast," +
+            "       serials.object_id," +
+            "       last.object," +
+            "       last.pkey " +
+            "FROM   serials " +
+            "       JOIN last " +
+            "              ON last.object_id = serials.object_id " +
+            "WHERE serials.atlast = 1", (rs, rowNum) -> new SerialEntry(
+            rs.getInt(1),
```

Comment:
If it is used for iniializing snapshot file, last will also contain deleted object entries ? Do we need to put sequence_id>0 check?

Reply:
not necessary, because when sequence_id is 0 atlast == 0 as well, so the WHERE clause will filter them out. having said that there is no db constraint on that condition, so it might be worth adding. what do you think?

Reply:
No you are correct its not necessary.